sphinx/search/zh.py

```diff
@@ -2,6 +2,7 @@
 
 from __future__ import annotations
 
+import inspect
```

Comment:
Can we have this import in the function it is being used just to save a bit of import time (unless this is already imported by some other dependency)?

Reply:
Why use inspect here? It's a very heavy package, we can just use `.__file__`?

Reply:
I wanted to use a public API but maybe it's an overkill (my bad since it was my suggestion :')).

Reply:
So just revert to 46753b903d29fb3d080e7c260c8020e24e5bb663 and dynamically get the dict file using `jieba.DEFAULT_DICT_NAME` is enough, I think.

Reply:
> Why use inspect here? It's a very heavy package, we can just use .__file__?

Should I modify the code to meet it?

Reply:
Let's just use the dunder attribute directly. Sorry for the back and forth

Reply:
> Let's just use the dunder attribute directly. Sorry for the back and forth

Done! Please review again.